### PR TITLE
fix computeScrollOffsets for IE11

### PR DESCRIPTION
**Pull request by rosivanov**
Related issue #160 
@yowainwright Hi! Seems i have fixed `stickyBitStickyOffset` option for IE11. At least now it works properly in ie.
Tested in IE11, Chrome 62, FFox 57. Unfortunately i have not opportunity to test it in Safari.
Can you please consider this PR and accept if it's ok.

### CODE REVIEW

File: src/stickybits.js [hunk 1]
```diff
@@ -199,8 +199,13 @@ Stickybits.prototype.computeScrollOffsets = function computeScrollOffsets(item)
     stickyStart = parent.getBoundingClientRect().top - scrollElOffset
   }
   it.offset = scrollElOffset + p.stickyBitStickyOffset
+  let stickyStop = (stickyStart + parent.offsetHeight) - (it.el.offsetHeight - it.offset)
+  if ( iw ) {
```
Comment (yowainwright) [inline src/stickybits.js]: Why is the `is window` check needed here?

File: src/stickybits.js [hunk 2]
```diff
@@ -199,8 +199,13 @@ Stickybits.prototype.computeScrollOffsets = function computeScrollOffsets(item)
...
+	  stickyStart = stickyStart - it.offset
+	  stickyStop = stickyStart + parent.offsetHeight - it.el.offsetHeight
```
Comment (yowainwright) [inline src/stickybits.js]: Is it only the `it.offset` that is creating a problem for `stickyStart`? 

Why is `stickyStop` adjusted?

**Comment by yowainwright** [comment]
It seems like this 1 line (based on your awesome work 🙏) is what you think is needed:
```js
it.stickyStart = stickyStart - it.offset
```
Is that right? Please tell me why or why not in a sentence or two. THANKS!

**Comment by rosivanov** [comment]
@yowainwright Hi! Thanks for CR. I will give replies later cause now i'm away from computer.

**Comment by rosivanov** [comment]
@yowainwright Hi Jeff! You are totally right. I have added extra & unnecessary code, so i rewrited it.
Please take a look to this commit  https://github.com/dollarshaveclub/stickybits/pull/162/commits/bb012ef38531cae856007e7b48552bd9491507c4
It should fix #160 & #159 Tested in IE11 also.

**Comment by yowainwright** [comment]
@rosivanov great work! See my comment [here](https://github.com/dollarshaveclub/stickybits/pull/164).

For future reference, rebasing is optimal! 🙏

**Pull request closed without merging**